docs/source/conf.py

```diff
@@ -113,7 +113,7 @@
     # torch.distributed.run
     "config_from_args",
     "determine_local_world_size",
-    "get_args_parser",
+    "create_args",
```

Comment:
No new entry should be added to this file in general.
Functions that are public API must be properly documented.

Reply:
This is a list for coverage_ignore_functions, which are not expected to be exposed to users.

Reply:
Functions that should not be exposed should not be public. You can make that function private (by making it start with an _)  to avoid having to add it to this list.

Reply:
Sure